src/sparseml/pytorch/sparsification/pruning/modifier_pruning_obs.py

```diff
@@ -388,7 +387,7 @@ def __init__(
         fisher_block_size: int,
         mask_type: str,
     ):
-        super().__init__(params)
+        super().__init__(params, dist_backend="nccl")
```

Comment:
This doesn't seem to be used anywhere. Guessing it's a dev relic?

Reply:
Specifying `dist_backend` enables creation of `self._dist_group`. This results in initializing a separate group for scorer. Without this, `self._dist_group` is set to None.

Reply:
Ah yeah I misread the code